Pirate/Program.cs

```diff
@@ -0,0 +1,190 @@
+                }
+
+                else if (answer == "yes")
+
+                {
+                    Console.WriteLine("PEERFECT LET'S BEGIN!!");
+                }
+            }
+            while (answer != "yes");
+
+
+
+            Console.WriteLine("I kneeeew yeee had it in ya!");
+
+            string answertwo;
+
+            do
+            {
+                answertwo = GetUserResponse("Would you like to search for buried treasure? Yes or No?");
+                Console.Clear();
+
+                if (answertwo == "no")
+                {
+
+                    Console.WriteLine("Ya want to stay at home and fiddle yer thumbs?!");
+                }
+                else if (answertwo == "yes")
+                {
+                    Console.WriteLine("Buried treasure it is!");
+                }
+
+            }while (answertwo != "yes");
+
+
+            Console.WriteLine("Your an adeventure lubber like I!");
+
+            string answerthree;
+            do
+            {
+
+                Console.WriteLine("AAARR I got two adventures on my horizon. Would you like to begin your journey for treasure?");
+                Console.WriteLine("Or would yee rather go whale hunting for supplies first?");
+               answerthree = GetUserResponse("Reply: whale hunting: or: begin journey");
+                Console.Clear();
+
+                if (answerthree == "whale hunting")
+                {
+                    Console.WriteLine("Excellent choice! Yee are well on your way to be a pirate!");
+                }
+                else if (answerthree == "begin journey")
+                {
+                    Console.WriteLine("Try again! We don't want to starve on our journey!");
+                }
+                else
+                {
+                    Console.WriteLine("speak up youngster! Do you not speech pirate??");
+                }
+            } while (answerthree != "whale hunting");
+
+
+            Console.WriteLine($"Alrighty then {newpiratename}, I got another test to see if you are truly ready to set sail.");
+
+            string answerfour;
+
+            do
+            {
+
+                Console.WriteLine("Yee got 4 measly choices so listen carefully! Choose yer letter!");
+
+                answerfour = GetUserResponse("Which direction is true pirate North? is it A.North on me compass? B.Wherever my captain says it is? C.The way the wind blows? or D.The fastest way to danger?");
+                Console.Clear();
+                if (answerfour == "A")
+                {
+                    Console.WriteLine("WRONG! You failed matey!Try again.");
+                }
+
+                else if (answerfour == "B")
+                {
+                    Console.WriteLine("What's wrong with you my friend!? Try again!");
+                }
+                else if (answerfour == "C")
+                { Console.WriteLine("THAT'S the SPIRIT!!"); 

```

Comment:
Consider putting this `Console.WriteLine()` on its own line instead of on the same line as the `{`